test/e2e/storage/drivers/in_tree.go

```diff
@@ -1134,6 +1144,9 @@ func (g *gcePdDriver) GetDriverInfo() *testsuites.DriverInfo {
 
 func (g *gcePdDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern) {
 	framework.SkipUnlessProviderIs("gce", "gke")
+	if pattern.FeatureTag == "sig-windows" {
```

Comment:
I think you will need to add a check like this for the CSI GCE PD driver config [here](https://sourcegraph.com/github.com/kubernetes/kubernetes/-/blob/test/e2e/storage/drivers/csi.go#L308)

Reply:
fixed